Header header

bool   hasSoundInEvt
bool   hasSoundFile
uint8  loud_level
int8   soundDirection 
string soundKinds
string txt_stt               #result of stt(from cloud or local stt)

